openwisp_notifications/base/models.py

```diff
@@ -151,6 +153,18 @@ def action_object(self):
     def target(self):
         return self._related_object('target')
 
+    @property
+    def redirect_view_link(self):
+        return _get_absolute_url(
+            '{redirect_view}?target_url={target}'.format(
+                redirect_view=reverse(
+                    f'{self._meta.app_label}:notification_read_redirect',
```

Comment:
app_label and URL namespace are two different concepts, do not mix them.
Let's use always `notifications` as URL namespace, see https://github.com/openwisp/openwisp-notifications/commit/5bb699dabf50a56bc370ec9262ceeb4da74201c0 to know why.